app.php

```diff
@@ -8,10 +8,8 @@
 
 $app = new Silex\Application();
 
-/**
- * Application parameters
- */
-$app['params'] = yaml_parse(file_get_contents(__DIR__.'/config/parameters.yml'));
+/** @var \Core\Entity\AppParameters $app['params'] */
+$app['params'] = new \Core\Entity\AppParameters(__DIR__.'/config/parameters.yml');
```

Comment:
is it OK to pas a filepath to initialise a Class??

Reply:
Yes :)